com.civicrm.hrjobroles/js/test/filters/option-group-filter_test.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-roles/filters/option-group-filter',
+    'job-roles/filters/filters',
+], function(angular, moment) {
+  'use strict';
+
+  describe('Using getActiveGroup filter', function() {
+    var $filter, locations, getActiveGroup;
+
+    beforeEach(module('hrjobroles.filters'));
+    beforeEach(inject(['$filter', function(_$filter_) {
+        $filter = _$filter_;
+        getActiveGroup = $filter('getActiveGroup');
+      }
+    ]));
+
+    it('should define getActiveGroup filter ', function() {
```

Comment:
`it` titles should never contain "should", they must always contain statements in the imperative mood
```js
it('should do something', function () {}); // wrong
it('does something', function () {}); // correct
```
in your case it can simply be
```js
it('is defined', function () {});